src/main.ts

```diff
@@ -1,10 +1,15 @@
 import {NestFactory} from '@nestjs/core';
-import {AppModule} from './modules/app/app.module';
+import {AppModule} from './app.module';
 import {setupSwagger} from './swagger';
+import * as passport from 'passport';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   setupSwagger(app);
+
```

Comment:
```suggestion
app.useGlobalPipes(new ValidationPipe());

```
npm i --save class-transformer